### ISSUE TITLE
Extend pre/post processing docs with real life examples

### ISSUE THREAD

**Issue by Siegrift** [api3dao/api3-docs#958]

**Comment by dcroote** [comment]
The [pre-processing](https://github.com/api3dao/airnode/tree/master/packages/airnode-examples/integrations/coingecko-pre-processing) and [post-processing](https://github.com/api3dao/airnode/tree/master/packages/airnode-examples/integrations/coingecko-post-processing) integrations in the `airnode-examples` monorepo provide working examples. Also, #1016 introduced docs links to these examples.

**Comment by Siegrift** [comment]
Yeah, but I meant it that the explantion in docs could start by desribing these use cases before dwelling into the confiugration and the implementation details.
